renku/core/incubation/graph.py

```diff
@@ -113,6 +113,9 @@ def process_datasets(commit):
     elif client.has_graph_files() or client.has_datasets_provenance():
         raise errors.OperationError("Graph metadata exists. Use ``--force`` to regenerate it.")
 
+    # database = Database.from_path(path=client.database_path)
```

Comment:
I guess we can completely remove this. I remember I had to add it because I got some errors otherwise but that seems to be fixed